modules/tide_edit_protection/src/Form/TideEditProtectionForm.php

```diff
@@ -0,0 +1,165 @@
+<?php
+
+namespace Drupal\tide_edit_protection\Form;
+
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\Core\Entity\EntityTypeBundleInfo;
+use Drupal\Core\Entity\EntityTypeManager;
+use Drupal\Core\Form\ConfigFormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Tide edit protection form.
+ */
+class TideEditProtectionForm extends ConfigFormBase {
+
+  /**
+   * Entity Type Manger.
+   *
+   * @var \Drupal\Core\Entity\EntityTypeManager
+   */
+  protected $entityTypeManger;
+
+  /**
+   * Entity Type Bundle.
+   *
+   * @var \Drupal\Core\Entity\EntityTypeBundleInfo
+   */
+  protected $entityTypeBundle;
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId() {
+    return 'tide_edit_protection_form';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function getEditableConfigNames() {
+    return ['tide_edit_protection.form'];
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function __construct(ConfigFactoryInterface $config_factory, EntityTypeManager $entityManager, EntityTypeBundleInfo $bundleInfo) {
+    parent::__construct($config_factory);
+    $this->entityTypeManger = $entityManager;
+    $this->entityTypeBundle = $bundleInfo;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container) {
+    return new static($container->get('config.factory'), $container->get('entity_type.manager'), $container->get('entity_type.bundle.info'));
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state) {
+    $config = $this->config('tide_edit_protection.form');
+    $entity_types = $entity_bundles = [];
+    $_entity_types = $this->entityTypeManger->getDefinitions();
```

Comment:
This naming convention is a little confusing. Can you improve the specificity of the variable and remove the prefix `_`? Same for `$_entity_bundles`

Reply:
Thanks @GROwen , changed.

Reply:
To confirm, no Yoda conditionals. Thanks for changing.